cpp/ql/src/semmle/code/cpp/security/TaintTrackingImpl.qll

```diff
@@ -1,4 +1,8 @@
 /**
+ * DEPRECATED: we now use `semmle.code.cpp.ir.dataflow.DefaultTaintTracking`,
+ * which is based on the IR but designed to behave similarly to this old
+ * libarary.
```

Comment:
@jbj I want to draw your attention to this.  I believe this file is the pre-defaulttainttracking version of the security taint library, which is no longer in use and should not be used except for comparison.  I wanted to add a warning to avoid confusion (e.g. when I'm trying to remember which of these files is which).  We should probably delete the file entirely at some point, but not until we've removed the remaining discrepancies.

Reply:
Thanks for the heads up. The next step is probably to rename this file into `LegacyTaintTracking.qll` or something like that that's more telling about its current role.

I think we can delete this file _before_ we've removed all the remaining discrepancies, as long as the new library is a clear enough improvement that nobody should be using the old one for any purpose. Deprecating the old library is a good first step.